apps/editors/models.py

```diff
@@ -395,6 +396,21 @@ def award_points(cls, user, addon, status, **kwargs):
         """
         event = cls.get_event(addon, status, **kwargs)
         score = amo.REVIEWED_SCORES.get(event)
+
+        try:
+            vq = ViewQueue.objects.get(
+                addon_name=addon.name,
+                addon_slug=addon.slug,
+            )
+
+            if vq.waiting_time_days > 10:
+                days_over = vq.waiting_time_days - 10
+                bonus = days_over * REVIEWED_OVERDUE_BONUS
+                score = score + bonus
+        except ViewQueue.DoesNotExist:
+            # If the object does not exist then we simply do not add a bonus
```

Comment:
We write comments as full sentences, so first letter is capitalized, and it ends with a `.`